js/gated-content/src/views/DurationPage.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="gated-content-duration-page">
+    <div v-if="loading" class="text-center">
+      <Spinner></Spinner>
+    </div>
+    <div v-else-if="error">Error loading</div>
+    <template v-else>
+      <div class="gated-containerV2 my-40-20 px--20-10 title-wrapper">
+        <div>
+          <div class="title-inline cachet-book-32-28">{{ duration.attributes.name }}</div>
+          <AddToFavorite
+            :id="duration.id"
+            :type="'taxonomy_term'"
+            :bundle="'gc_duration'"
+          ></AddToFavorite>
+        </div>
+      </div>
+
+      <div v-for="component in componentsOrder" :key="component">
```

Comment:
As we have durations only for videos -- seems no sense to use the roll around the componentsOrder here